### ISSUE TITLE
Add to testing framework: NEON

### ISSUE THREAD

**Issue by darianmiller** [hydrobyte/TestJSON#4]
Source: https://github.com/paolo-rossi/delphi-neon
More info: https://blog.paolorossi.net/json-serialization-with-neon-part-1/

**Comment by hydrobyte** [comment]
Hi.
Thanks for your suggestion.
I've started the Delphi version and I'll try to include `Neon`.

**Comment by hydrobyte** [comment]
Hi.

Neon - JSON is a serialization library for Delphi and internally it uses System.JSON.

I think this is not the case here.

**Comment by darianmiller** [comment]
Yes, NEON uses the internal classes but adds enhanced functionality. Curious to see how much it slows down the tests compared to the built-in classes.  Also to ensure it doesn't break any tests.

**Comment by darianmiller** [comment]
NEON seems to be pretty popular as it has some unique functionality.

**Comment by hydrobyte** [comment]
I think NEON just improve something like helpers into `TJSONUtils` class.
It seems to me that Add, Load, Save, Find and Parse will still use System.JSON.
If you have examples that contradict this feeling, please report here to me.
